lib/Models/TableStyle.js

```diff
@@ -53,6 +53,12 @@ var TableStyle = function(options) {
    */
   this.dataVariable = options.dataVariable;
 
+  /**
+   * Whether all variables have been unselected
+   * @type {String}
+   */
+  this.allVariablesUnactive = options.allVariablesUnactive;
```

Comment:
I'm not sure if this is better as a computed value, I wanted to avoid any more interactions between tableStyle <-> tableStructure 😵 and thought it'd be simplest to set this on serialization.